docs/resources/observability_pipeline.md

```diff
@@ -0,0 +1,350 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_observability_pipeline Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Pipelines resource. This can be used to create and manage Datadog pipelines.
+---
+
+# datadog_observability_pipeline (Resource)
+
+Provides a Datadog Pipelines resource. This can be used to create and manage Datadog pipelines.
```

Comment:
```suggestion
Provides a Datadog pipelines resource. This can be used to create and manage Datadog pipelines.
```

Reply:
Do you mean Observability Pipelines? There is no product named Datadog Pipelines as far as I know.

Reply:
Right, thanks, updated.